Makefile

```diff
@@ -292,7 +292,7 @@ start-test-app:
 
 # run tests
 .PHONY: test
-test: license lint
```

Comment:
@kuritka why did we remove local invocation?

Reply:
I supposed you didn't want to check license during a test. Should I amend back ?

Reply:
@kuritka yep, I meant to amend it with CI, not replace. So CI would catch the missing license in case the PR was created without running local tests

Reply:
@ytsarev exactly